### ISSUE TITLE
flex-grow decimal value isn't respected in Safari

### ISSUE THREAD

**Issue by realseanp** [philipwalton/flexbugs#182]
I have a column layout with two containers stacked on top of one another. The wrapper element is set to a height of `100vh`. The first child is set to `flex-grow: 0.8` this property works in firefox and chrome, but Safari only seems to respect 0 and 1, so in Safari the container is too big. Still looking for a work around.

```
<div class="parent"> // height: 100vh
    <div class="child-1">...</div> // flex-grow: 0.8, not respected in Safari
    <div class="child-2">...</div>
</div>
```

**Comment by realseanp** [comment]
I fixed the issue by ditching `flex-grow: 0.8` and using a percentage. I also changed the position to absolute. Here's an example.

```css
.parent {
    height: 100vh;
    width: 100%;
    position: relative;
}

/* This used to have the property flex-grow: 0.8 */
.child-1 {
    position: absolute;
    width: 100%;
    height: 80%;
}

.child-2 {
   position: absolute;
   bottom: 0;
   width: 100%
}
```

**Comment by philipwalton** [comment]
Related to https://github.com/philipwalton/flexbugs/issues/116.
